renaissance-core/src/main/java/org/renaissance/ValidationException.java

```diff
@@ -0,0 +1,45 @@
+package org.renaissance;
+
+public class ValidationException extends RuntimeException {
```

Comment:
Nice design.
But shouldn’t we have a class for the Exception and another one for the Validator ?

I find unnatural to call `ValidationException.throwIfNotEqual()`, and, IMHO, I think that it would be nicer to do `Validator.throwIfNotEqual()`.

This API is not exposed in the benchmark code though, so it’s not critical.

Reply:
> I find unnatural to call `ValidationException.throwIfNotEqual()`, and, IMHO, I think that it would be nicer to do `Validator.throwIfNotEqual()`.

That makes sense but I would wait how it aligns with #81, i.e. whether this even survives ;-).